cmd/grype/cli/legacy/root.go

```diff
@@ -192,6 +198,16 @@ func setRootFlags(flags *pflag.FlagSet) {
 		"platform", "", "",
 		"an optional platform specifier for container image sources (e.g. 'linux/arm64', 'linux/arm64/v8', 'arm64', 'linux')",
 	)
+
+	flags.StringArrayP(
+		"vex", "", []string{},
+		"a list of VEX documents to consider when producing scanning results",
+	)
+
+	flags.BoolP(
+		"vex-filter", "", true,
```

Comment:
we could probably drop this flag on the first pass (where if the user passes a vex doc then it implicitly filters)

Reply:
from the conversation: this was here to enable filtering or enable adding matches based on vex. Since the ignore rules already covered in the application configuration we can try to at least cover the addition of matches based on vex statements:

```
GRYPE_VEX_ADD=affected,under_infestigation     # default "", or allow "true" which means just "affected"
```

So by default grype would filter by default when provided a vex document, but the user would need to opt-into augmenting/adding matches based on vex documents.

Reply:
(consider this config example to be not finalized / illustrative)

Reply:
OK, we now have the results augmentation feature implemented see [my comment](https://github.com/anchore/grype/pull/1397#issuecomment-1689235897) below.